code/modules/assembly/proximity.dm

```diff
@@ -11,11 +11,19 @@
 
 	var/scanning = FALSE
 	var/timing = FALSE
-	var/time = 10
+	COOLDOWN_DECLARE(timing_cd)
```

Comment:
This doesn't change its behavior, right? Was this original variable in seconds, ds, or processing ticks?

Reply:
I just did some extra testing. As far as I can tell, the original sensor cooldown is 10 ticks. I'm almost positive I ported the behavior correctly, both for the sensor cooldown and the arming timer. There's also "cooldown", which is not related to the timing countdown, meant to prevent spamming the sensor, which is part of the parent proc `/obj/item/assembly/proc/process_cooldown()`, and was already on a timer so the behavior there shouldn't have changed either.